### PR TITLE
I'm finalizing the package.json. Stand the dependence

### PR DESCRIPTION
**Pull request by varbachakov**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,7 +1,4 @@
 {
-  "presets": [
-    "react",
-    "env",
-    "stage-1"
-  ]
-}
+  "presets": ["env", "stage-0", "react"]
+
```
Comment (ubcent) [inline .babelrc:3]: Unnecessary line break

File: package.json [hunk 2]
```diff
@@ -22,31 +22,32 @@
...
     "babel-core": "^6.3.15",
     "babel-eslint": "^7.2.3",
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1",
-    "babel-register": "^6.8.0",
-    "chai": "^3.5.0",
-    "enzyme": "^2.3.0",
+    "babel-preset-stage-0": "^6.24.1",
```
Comment (ubcent) [inline package.json:31]: Do we really need stage 0?

File: package.json [hunk 3]
```diff
@@ -22,31 +22,32 @@
...
   "devDependencies": {
+    "autoprefixer": "^9.1.0",
     "babel-core": "^6.3.15",
     "babel-eslint": "^7.2.3",
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1",
-    "babel-register": "^6.8.0",
-    "chai": "^3.5.0",
```
Comment (ubcent) [inline package.json:32]: Why do you remove that?

**Review by ubcent — CHANGES_REQUESTED** [review]

**Comment by varbachakov** [comment]
Modification

**Pull request closed without merging**